docs/rules/README.md

```diff
@@ -11,6 +11,7 @@
 * [no-nested-tests](no-nested-tests.md) - disallow tests to be nested within other tests
 * [no-pending-tests](no-pending-tests.md) - disallow pending/unimplemented mocha tests
 * [no-return-and-callback](no-return-and-callback.md) - disallow returning in a test or hook function that uses a callback
+* [no-setup-in-describe](no-setup-in-describe.md) - disallow calling functions and dot operaters directly in describe blocks
```

Comment:
Dot operators? Do you mean member expressions? If so, please also consider that member expression could be written as `foo[bar]`.

Reply:
Ooh, yes, on it!
 - [ ] handle bracket notation (`foo[bar]`)